index.d.ts

```diff
@@ -18,8 +23,8 @@ export interface Options {
   testNamePattern?: string;
   testFileNamePattern?: string;
   reporters?: string[];
-  // custom args if specified will supercede any auto arguement logic
-  args?: string[];
+  /** either to append or replace the Runner process arguments */
+  args?: RunArgs;

```

Comment:
Do we care about introducing a breaking change? If yes, we could also use `args?: RunArgs | string[];`.

One thing I noticed with `RunArgs` is that it will read as `options.args.args`. Maybe it would be better to have the `replace` property as part of `Options`? We could also call it something like `SkipDefaultArgs`.

Reply:
yeah, I thought about `args?: RunArgs | string[]` but we still have the `options.args.args` situation while trying to get flow to work with this kind of type... Regarding moving the "SkipDefaultArgs" flag to the options itself, it won't achieve the goal that this flag should always accompany the args that replace it...therefore, my gut feeling is it might not be worthy to do a breaking change right now, as none of the alternatives is perfect either... maybe revisit this when we migrate the rest of the package to typescript.